.gitignore

```diff
@@ -37,19 +42,13 @@ npm-debug.log
 yarn-error.log
 testem.log
 /typings
-
-# System Files
-.DS_Store
-Thumbs.db
```

Comment:
Might as well keep it, OSX/Windows thumbnails, if you ever open the project in regular file explorer, then this will be gen'd

Reply:
Ah those got moved, if you feel like playing diff golf to ease reading and spot additions/deletions, not a big deal otherwise